docs/usage/self-hosted-configuration.md

```diff
@@ -184,6 +184,10 @@ Note that if `commitMessagePrefix` or `semanticCommits` values are defined then
 
 ## onboardingConfig
 
+## onboardingConfigFileName
+
+If set to one of the valid [config file names](https://docs.renovatebot.com/configuration-options/), the onboarding PR will create a configuration file with the provided name instead of `renovate.json`. Falls back to `renovate.json` if the name provided is not valid.
```

Comment:
```suggestion
If set to one of the valid [config file names](https://docs.renovatebot.com/configuration-options/), the onboarding PR will create a configuration file with the provided name instead of `renovate.json`.
Falls back to `renovate.json` if the name provided is not valid.
```